requirements/production.txt

```diff
@@ -0,0 +1,184 @@
+#
```

Comment:
How did we generate this file?

Reply:
just copied from base.txt

Reply:
discussed with sobhan and followed the current way by installing requirements from base.txt and pip.txt